data/events/scripts/party.lua

```diff
@@ -1,18 +1,19 @@
 function Party:onJoin(player)
-	return true
+	if hasEventCallback(EVENT_CALLBACK_ONJON) then return EventCallback(EVENT_CALLBACK_ONJON, self, player) else return true end
```

Comment:
who is jon? 😛

Reply:
will fix that, thanks for pointing it out!